kodi_voice/kodi.py

```diff
@@ -50,6 +50,14 @@ def sanitize_name(media_name, remove_between=False, normalize=True):
   name = name.strip()
   return name
 
+def sanitize_channel(name, lang='en'):
+  name = name.lower()
+  name = name.replace("+", " plus ")
+  name = re.sub(r"\sch\s", " channel ", name)
+  name = re.sub("^channel", "", name)
+  name = re.sub(r"(?<=\D)(?=\d)|(?<=\d)(?=\D)", " ", name)
+  name = words2digits(name, lang=lang)
```

Comment:
It might be easier just to call santize_name from within sanitize_channel?

I put in the words2digits here because I couldn't get the matching to work reliably otherwise, but I've probably misunderstood something somewhere...

Channels on the backend often have "number words" in the name e.g. "BBC One HD".  When you say "ask kodi to switch to channel BBC One HD", you seem to get given the string "BBC 1 hd" in the JSON from amazon.  I think it's this string that gets words2digits applied to it by the fuzzy matcher?  

Without this words2digits in sanitize_channel the simple match fails (BBC 1 hd != BBC One HD) and I kept getting given CBBC HD by the digits2roman fuzzy match.  What I've tried to do in sanitize_channel is make all the channel names more similar to the string you get from amazon so that the simple match stands a chance.